packages/components/src/custom-gradient-picker/utils.js

```diff
@@ -174,3 +178,33 @@ export function getMarkerPoints( gradientAST ) {
 		};
 	} );
 }
+
+export function getLinearGradientRepresentationOfARadial( gradientAST ) {
+	return serializeGradient( {
+		type: 'linear-gradient',
+		orientation: HORIZONTAL_GRADIENT_ORIENTATION,
+		colorStops: gradientAST.colorStops,
+	} );
+}
+
+export function getGradientParsed( value ) {
+	let hasGradient = !! value;
+	// gradientAST will contain the gradient AST as parsed by gradient-parser npm module.
+	// More information of its structure available at.
```

Comment:
> More information of its structure available at:
I think we're missing a link in this comment.

Reply:
Nice catch 👍